### DISCUSSION TITLE
Openai-model-fine-tune-about-ZKP

### DISCUSSION THREAD

**Discussion by Demian101** [zkp-co-learning/zkp-co-learn#69] [I've got an idea!]
搞个 zk 的问答知识库？ 可能会比较精准一些，回答问题什么的，省的整天在数学问题上胡说八道。。

http://zkpod.ai/

有没有想一起搞一个的（这个除了内容，跟 zk 倒是没啥关系）：
1. 需要搞一个 openai 的高级 API （众筹买一个？）
2. 处理下模型的 fine-tuning
3. 处理下前后端什么的

**Comment by luckyyang** [comment]
感觉可以搞啊
